testsuite/cluster-test/src/suite.rs

```diff
@@ -79,11 +79,31 @@ impl ExperimentSuite {
         Self { experiments }
     }
 
+    fn new_land_blocking_compat_suite(cluster: &Cluster) -> Result<Self> {
+        let count: usize = match env::var("BATCH_SIZE") {
+            Ok(val) => val.parse().expect(&format!("Invalid BATCH_SIZE {}", val)),
```

Comment:
So .expect here will just panic, since you returning Result, you can just return proper error instead:

`val.parse().map_err(|e| format_err!("Failed to parse BATCH_SIZE {}: {}", val, e)?`

Reply:
noted!